make/proto.mk

```diff
@@ -4,38 +4,40 @@
 ###                                Protobuf                                 ###
 ###############################################################################
 
-containerProtoVer=v0.7
-containerProtoImage=tendermintdev/sdk-proto-gen:$(containerProtoVer)
-containerProtoGen=cheqd-node-proto-gen-$(containerProtoVer)
-containerProtoFmt=cheqd-node-proto-fmt-$(containerProtoVer)
-containerProtoGenSwagger=cheqd-node-proto-gen-swagger-$(containerProtoVer)
-containerPulsar=cheqd-node-pulsar-gen-$(containerProtoVer)
+DOCKER := $(shell which docker)
+containerProtoVer=0.14.0
+containerProtoImage=ghcr.io/cosmos/proto-builder:$(containerProtoVer)
+protoImage=$(DOCKER) run --rm -v $(CURDIR):/workspace --workdir /workspace $(containerProtoImage)
+# containerProtoGen=cheqd-node-proto-gen-$(containerProtoVer)
+# containerProtoFmt=cheqd-node-proto-fmt-$(containerProtoVer)
+# containerProtoGenSwagger=cheqd-node-proto-gen-swagger-$(containerProtoVer)
+# containerPulsar=cheqd-node-pulsar-gen-$(containerProtoVer)
```

Comment:
Is any further action required for swagger, pulsar generation included in any the above old logic?

Looks like the proto-builder is sufficient, but let's make sure that's the case.

If strong yes, let's remove these lines.

Reply:
Done